src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Triggers/ServiceBusTriggerInput.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+
+using Microsoft.Azure.ServiceBus;
+using Microsoft.Azure.ServiceBus.Core;
+
+namespace Microsoft.Azure.WebJobs.ServiceBus
+{
+    // The core object we get when an ServiceBus is triggered. 
+    // This gets converted to the user type (Message, string, poco, etc) 
+    internal sealed class ServiceBusTriggerInput
+    {
+        // If != -1, then only process a single message in this batch. 
+        private int _selector = -1;
```

Comment:
This whole int selector thing is weird and we should remove it. I realize it's a copied pattern from EventHub. It's only used in the GetSingleMessage method, and that method should go away (see comments there).